### PR TITLE
Add high-level API reference to README

### PR DESCRIPTION
**Pull request by Chris-hughes10**
* Updates the README with a reference to a high-level API built on top of Accelerate
* Fixes a small type in the notebook launcher's log statement

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Thanks for your PR! I was more thinking of having a separate section named "Frameworks using Accelerate" where you could put the same content. There is also kornia using Accelerate internally in their trainer I would add there :-)

**Comment by Chris-hughes10** [comment]
> Thanks for your PR! I was more thinking of having a separate section named "Frameworks using Accelerate" where you could put the same content. There is also kornia using Accelerate internally in their trainer I would add there :-)

Sure thing, I can create an additional section, migrate the content, and will also look into and add Kornia.

**Comment by Chris-hughes10** [comment]
@sgugger I have updated the README, hopefully that is closer to what you had in mind. Would you also like to me add a corresponding section to the docs?

File: README.md [hunk 1]
```diff
@@ -220,6 +227,8 @@ Then, you will need to install PyTorch: refer to the [official installation page
 pip install accelerate
 ```
 
+
+
```
Comment (sgugger) [inline README.md:231]: No need for those two new lines here :-)

**Review by sgugger — APPROVED** [review]
Thanks a lot for adding this new section!

File: README.md (hunk 1 above)
Comment (Chris-hughes10) [reply README.md]: Removed.

**Pull request merged**